python/src/puml/helpers/requirements_helpers.py

```diff
@@ -0,0 +1,132 @@
+from typing import List, Optional
```

Comment:
Need your intro docstring
```suggestion
"""
Helper methods for extracting and sorting pertinent model data for use in generating requirements
diagrams in a PUML format.
"""
from typing import List, Optional
```

Reply:
Added